jobs/enthought_web-developer.html

```diff
@@ -0,0 +1,41 @@
+---
+title: Full Stack Web Developer 
+company: Enthought
+url: http://www.enthought.com/company/careers/
+location: Cambridge, UK
```

Comment:
@salimfadhley thanks for the review! I'll push some updates in a separate PR as this one is already merged.